### DISCUSSION TITLE
GitHub Global Campus and Codespaces for Teachers are live!

### DISCUSSION THREAD

**Discussion by github-education** [community/community#17765] [GitHub Education]
:wave: Hey teachers, professors, and educators!

As the end of one semester rolls into the next, GitHub Education is proud to announce the launch of two new products: GitHub Global Campus and Codespaces for Teachers. 

Navigating all of the GitHub benefits for teachers has been cumbersome in the past and the conversation has always been to bring all of the features and actions all into a singular home.  Announcing Global Campus, a portal to upgrading GitHub organizations to GitHub Teams, managing active GitHub Classrooms, connecting with the teacher community with exclusive GitHub Discussions, and even requesting swag for your classroom - all in one place!

Along with the announcement of Global Campus, teachers using GitHub Classroom now get access to GitHub’s groundbreaking browser-based online IDE Codespaces at no cost. The free allowance is estimated to be enough for a class of 50 with 5 assignments per month, on a 2 core machine with 1 codespace stored per student, which is more than enough to get your students coding like pros in an environment that brings out the best in their skills and allows you to teach seamlessly.

Ready to gain access and explore?  If you’re a teacher, you can join Global Campus by completing a short application for [teacher benefits](https://education.github.com/discount_requests/teacher_application?utm_source=2022-05-09-Edu-blog). Once accepted, you will be officially welcomed as a Global Campus teacher. You can then access Global Campus anytime at https://education.github.com/. You will also be able to enable Codespaces in your GitHub Classroom assignments. Once you create an assignment with Codespaces and share it with your students, your students will have a new repository to complete their work with a Codespaces development environment. If you haven’t joined already, Global Campus and Codespaces is the latest and greatest reason to become a GitHub verified teacher!

If you are already a verified teacher, you can also go to https://education.github.com, sign in, and you will automatically be redirected to the Global Campus experience.

Want to dive into more details?  Check out our [announcement](https://github.blog/2022-05-09-prepare-for-next-semester-with-github-global-campus-and-codespaces/)!  Got questions? Feedback? Hype? Leave a comment below!

**Comment by suprstarrd** [comment]
Well, I definitely just found something I shouldn't have (relating to this) and I'm a student 😝 I sincerely apologize to whoever has to read bug report support tickets

**Reply by juanpflores** [reply]
Thanks for reporting 🙌🏼 Our team is reviewing the ticket

**Comment by WJC-POG** [comment]
I've applied almost 20 times but keep getting rejections. I've tried uploading my work documents, course schedule, and employment contract. I am no longer sure what other official documents I need to provide to prove that I am a university teacher.
